### PR TITLE
add server and dockerfile

### PR DESCRIPTION
**Pull request by securims**

### CODE REVIEW

File: server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.16-alpine
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY *.go ./
+RUN go build -o /server
+
+EXPOSE 8080
```
Comment (truskovskiyk) [inline server/Dockerfile:12]: could we switch a user after we install deps?

File: server/instructions.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Container is available at ghcr.io/securims/ml-prjctr
```
Comment (truskovskiyk) [inline server/instructions.md]: would be nice to have a README with full instructions, how to build, how to push, etc

**Review by truskovskiyk — APPROVED** [review]
@securims amazing!

File: server/Dockerfile (hunk 1 above)
Comment (securims) [reply server/Dockerfile:12]: done

File: server/instructions.md (hunk 2 above)
Comment (securims) [reply server/instructions.md]: done

**Review by truskovskiyk — APPROVED** [review]
@securims great job!

**Pull request merged**